### DISCUSSION TITLE
On which US coast can Redwood trees be found?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#250] [Galaxy Trivia] [answered]
**Hint:** OR CA

**Comment by thuongtruong1009** [comment, marked as answer]
Redwood trees can be found on the west coast of the United States, specifically along the Pacific Coast in California and Oregon. The coastal redwoods (Sequoia sempervirens) are the tallest trees in the world and can grow up to 379 feet (115.5 meters) in height. The redwood forests are a major tourist attraction and provide important ecological and economic benefits to the region.
